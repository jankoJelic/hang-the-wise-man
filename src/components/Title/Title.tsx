type Props = {
  text: string;
};

const Title: React.FC<Props> = ({ text }) => (
  <h1 className="font-bold text-2xl sm:text-4xl text-mainText text-center mb-8">
    {text}
  </h1>
);

export default Title;
