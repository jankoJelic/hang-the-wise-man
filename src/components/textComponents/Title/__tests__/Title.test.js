import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Title from "components/textComponents/Title";

it("renders correctly", () => {
  const tree = renderer.create(<Title text="Test" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct text", () => {
  const wrapper = render(<Title text="Test" />);

  expect(wrapper.getByText("Test")).toBeInTheDocument();
});
