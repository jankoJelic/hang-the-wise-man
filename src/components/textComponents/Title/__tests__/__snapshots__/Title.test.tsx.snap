// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  className="font-bold text-2xl sm:text-4xl text-mainText text-center mb-8"
>
  Test
</h1>
`;
